designs/administrator/future-openshift/saved-searches/index.md

```diff
@@ -0,0 +1,46 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Saved filters
+
+The Search page will be updated to allow users to save searches.
+
+## General flow
+
+![1](img/saved1.png)
+* If no filters have been saved, the **Saved filters** dropdown will have a small empty state.
+
+![2](img/saved2.png)
+* Users can save filters by clicking the **Save filter** button at the end of the filter chip row.
+
+![3](img/saved3.png)
+* Users will be able to enter a name for the saved filter in a modal.
+
+![4](img/saved4.png)
+* The **Saved filters** dropdown will update to show the saved filter name since it is the currently applied filter.
+
+![5](img/saved5.png)
+* Users can click the **Saved filters** dropdown to see all their saved searches.
+* Through the dropdown, users may also choose to edit their saved searches.
+
+![6](img/saved6.png)
+* In the **Edit saved filters** modal, users are able to edit the name of any previously saved filter.
+
+![7](img/saved7.png)
+* Using [inline edit](https://www.patternfly.org/v4/documentation/core/components/inlineedit), users can update and save a new filter name.
+* While the inline edit action is activated, the modal's **Save** button should be disabled.
+
+![8](img/saved8.png)
+* Users can also choose to remove a saved search by clicking the minus icon.
```

Comment:
I assume if the active filter is removed, page reverts back same as when one clicks on 'Clear Filter'.

Probably beyond this design, but I notice screenshots have the '3 selected' in the toolbar.  What are the thoughts around preserving the selected rows IF they are also in a newly selected saved filter?  I imagine it would be up to the app.   
Similar question, probably app specific, but I imagine the saved filters will be local to the particular browser (ie. saved in browser local storage/cookies).  Probably up to the app if they can save filters on server by user etc..

Reply:
That's a good question. @gdoyle1 I know you've done some work around bulk actions and selecting items from a list. What do you imagine would happen in the case above? Should we reset the selection if the user changes the filters? Or still filter down the list and update – for example they have 4 selected, then apply a filter, and now only have 2 selected.